keyboards/fleuron/readme.md

```diff
@@ -0,0 +1,16 @@
+# Fleuron
+
+![Fleuron](https://i.imgur.com/ejXBFRP.jpg "the fleuron keyboard")
+
+The Fleuron keyboard is a 6x16 ortho keyboard supporting right, left, or center numpad placement and rgb backlighting.
+
+Keyboard Maintainer: [James Underwood](https://github.com/ju0)  
+Hardware Supported: The Fleuron pcb, ATmega32u4 based controller
```

Comment:
This line needs a double space at the end, so that it's formatted correctly.  

Right now, it has a single space at the end, is it is merging the two line.